spec/factory_specs/subject_spec.rb

```diff
@@ -1,12 +1,24 @@
 require 'rails_helper'
+describe "Subject Factory" do
+  describe "Subject 'Further Education' Factory" do
+    let(:subject) { create(:futher_education_subject) }
 
-describe "Subject 'Further Education' Factory" do
-  let(:subject) { create(:futher_education_subject) }
+    it "created subject" do
+      expect(subject).to be_instance_of(Subject)
+      expect(subject).to be_valid
+      expect(subject.subject_name).to eq 'Further Education'
+      expect(subject.in?(Subject.further_education)).to be true
```

Comment:
How about using one-liners here?

```
let(:subject) { create(:further_education_subject) }

it { should be_instance_of(Subject) }
it { should be_valid }
its(:subject_name) { should eq 'Further Education' }
it { should be_in(Subject.further_education) }
```

If not, the last expect here could be this:

```
expect(subject).to be_in(Subject.further_education)
```

Reply:
out of scope, but yeah

Reply:
I'll sort it separately as there's a typo in further anyway

Reply:
Done